realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/UserFactory.java

```diff
@@ -16,18 +16,44 @@
 
 package io.realm.objectserver.utils;
 
+import java.util.UUID;
+
+import io.realm.Realm;
+import io.realm.RealmConfiguration;
 import io.realm.SyncCredentials;
 import io.realm.SyncUser;
+import io.realm.log.RealmLog;
 
+// Helper class to retrieve users with same IDs even in multi-processes.
```

Comment:
Should this store the user? `SyncUser.currentUser()` can get the current login user I think?